dbt/adapters/redshift/connections.py

```diff
@@ -178,18 +178,24 @@ def cancel(self, connection: Connection):
 
     @classmethod
     def get_response(cls, cursor: redshift_connector.Cursor) -> AdapterResponse:
+        # redshift_connector.Cursor doesn't have a status message attribute but
+        # this function is only used for successful run, so we can just return a dummy
         rows = cursor.rowcount
-        message = f"cursor.rowcount = {rows}"
+        message = "SUCCESS"
         return AdapterResponse(_message=message, rows_affected=rows)
 
     @contextmanager
     def exception_handler(self, sql):
         try:
             yield
         except redshift_connector.DatabaseError as e:
-            logger.debug(f"Redshift error: {str(e)}")
+            try:
```

Comment:
bug 1: this seems like the most expedient way to default to the exception object if there's any trouble accessing an inner message field

Reply:
Oof, so it's literally get the first error's "M"essage? They didn't happen to wrap that in anything?